docs/providers/auth0.md

```diff
@@ -26,7 +26,7 @@ You can override any of the options to suit your own use case.
 ## Example
 
 ```js
-import Auth0Provider from `next-auth/providers/auth0`
+import Auth0Provider from 'next-auth/providers/auth0'
```

Comment:
```suggestion
import Auth0Provider from "next-auth/providers/auth0"
```

Just so I can nitpick, I prefer double-quotes for strings. 😅 Sorry. :grimacing: 

We should be trying to be consistent about it throughout the docs, that would actually be more important than my own preference...

Do you know if Prettier can format code snippets in markdown files?

@ndom91?

Reply:
Haha good question, not sure. Idk how the backticks got in there, but I do use single quotes sometimes. Double quotes from now on ;)

Reply:
Yep [Prettier can format code snippets in Markdown files](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAJOBDAJnATgHSgF4TTDCB9AGQggGsAaCgAkh2YCMAbCMOgZ2YZccZgDMIuALYYY8LMwBUMWgEJF5KAAMdzAFb9CYgK5QwMAJbRmFnLAswAngAoAHgEpmwZiJjHcUMyuzAC+hDpamgAKFjwwAD4Agha4AA6SCagQ-oZQALR58UgFSPElhABSEAAWgQAiEHDxAMoA0gDi8QCMAMw9AByVGFCiAEqN8RUAYq3xPV0ALJp5zCtrqxvrW5ubmgDUzFQW-DCEzAcA7g7VQswA2lwWUHQAuszOAPQqH-wQUnAXap4ODuQgHYYKO7DCAwIG4ZjQOAvTRne7Q2F4BEjF5IZjMapyVJID4fOCuDBSVJcOAAOkgUmYAHIAKLkynU5iWGDUxmaGgiBkWVL8YwMrAQHjw-gOIT-GAMVjQfhwcxwPzw7BC45gJ4Ac2YcEeMBphAAwv4MBwHP5FVBlar1cwZK4LFJjIJcMd3QrlQpna73ZzDVxA5YzLZTDADcaQAwQBBUpYlchQMJcBALlFhAh+MgQBguBcMI5c3GOLgMHw1c1UpW9cgYLhjHA43ApBw4FgcFgqMNdcYMLq4FNJDI5PWUBhjCpYyACVIuAB1aoOOD8WtgODNHMOCwANwcjjzYH4pZAT2VuBgUQrupkyDEBeVcYMrgAQhWqzBmhS4EcRg+T4tiAr7NHq1IAIrGDCcCAVwz4gLWuCXnmMi4HQ4oXFAs6pJ6sCLrYsLIP0AAMca4RAyqLhWqR5rha54HusFxgAjtB8A3gmuaTvweQjJ2naziIbEpHAN6DveSCPvBwHKlIFgNk2sngXAUEwXBCEwJaBFYERSAAExxo2GCxHqpp-JJIBrgArLO7pwAAKpa3HSQhe7NgAklAdjfmAnqJok3nNE41IaXAIQhEAA) providing that they've got that snippet format (which this example does).

I'll close this PR as a Prettier rewrite of everything makes more sense.